definitions/pipelines/rnaseq_star.cwl

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "RNA-Seq alignment and transcript/gene abundance workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outSAMattrRGline:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    genomeDir:
+        type: Directory
```

Comment:
is this the directory path to the reference? If so, make that clear in the variable name

Reply:
changed it to stargenomeDir